lava-slave-install.md

```diff
@@ -0,0 +1,122 @@
+This is an howto install a lava-docker slave
+This howto is based on a Debian host.
+
+# Create a dedicated user
+The user will be lavadocker in our example. It must be in the docker group for doing docker operations.
+```
+useradd -G docker lavadocker
+```
+
+# Install all pre-requisites packages
+docker-compose need to be installed from PIP
+```
+apt-get install git python-pip
+```
+
+# Install docker-ce
```

Comment:
The docker instructions here are incomplete.  Please just link to the Docker documentation for debian installs: https://docs.docker.com/install/linux/docker-ce/debian/

Reply:
fixed